src/xdist/scheduler/each.py

```diff
@@ -103,6 +103,9 @@ def mark_test_complete(self, node, item_index, duration=0):
     def mark_test_pending(self, item):
         raise NotImplementedError()
 
+    def remove_pending_tests_from_node(self, node, indices):
```

Comment:
Some schedulers were missing this method that is used by `DSession`, if guess it's not called for them so raising `NotImplementedError`.